### ISSUE TITLE
elAPI show teams without owners

### ISSUE THREAD

**Issue by alexander-haller** [uhd-urz/elAPI#13]
Feature Request, 

elAPI show teams without owners

**Comment by mhxion** [comment]
Maybe this can be command/sub-command for the user management plugin?

**Comment by alexander-haller** [comment]
yes sure - generally it would be nice if we could unify our metadata and elabftw. Owner was one of those where it would be possible.

**Comment by alexander-haller** [comment]
https://github.com/elabftw/elabftw/issues/3454

**Comment by mhxion** [comment]
We discussed this in our weekly meeting. We mainly wanted to get the "owners" assigned to each team in the eLab server. We also keep track of owners in our internal metadata. We would like to avoid this duplicated information tracking in the future.

The main desired feature here is that we want to be able to add additional metadata to teams. Currently, there is no official way to do this. There are 2 workarounds to this that we explored:

- Adding (and maintaining) an external column in the eLabFTW database. The downside of this approach is that would require SQL knowledge to be able fetch/read the records.
- We can use the "Extra fields" of an experiment. I.e., we can use an experiment page as an information and metadata holder for each team, or, use a singe experiment page to store metadata about all teams.

<img width="1002" height="741" alt="Image" src="https://github.com/user-attachments/assets/eba52b74-4cc2-467c-ad96-2a86b1281881" />
